cgmanifest.json

```diff
@@ -8596,6 +8596,16 @@
         }
       }
     },
+    {
```

Comment:
May be good to run the update cgmanifest script one last time here. New packages liked `tuned` and packages moved from SPECS-EXTENDED don't seem to be here.

Reply:
They are in there, but some packages were promoted from SPECS-EXTENDED, rather than added directly (hence it not showing up in the cgmanifest diff) 

https://github.com/microsoft/CBL-Mariner/commit/21fac3d7e3410a46e68a11f8b504cdd5a59c3d5c#diff-45a0a38d395bf4427d0291d9c72bd99b57fa43e14b00e059ae170d657949f1f2

Reply:
Ah thanks for the clarification. I didn't see it in the github render but they are indeed there.